.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
+.idea
```

Comment:
```suggestion
```
I used to do this too, before I learned about setting up a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for these kinds of rules. I don't think this change is necessary.